api_keys.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we remove this file? We don't use the census api anymore (plus don't want to commit any API keys :) )

Reply:
yeah i thought that was weird -- git wouldn't let me pull from master b/c of a conflict but I can delete